docs/lib/geo/fragments/js/maps.md

```diff
@@ -16,14 +16,10 @@ Add the dependencies to your app:
 npm install -S maplibre-gl maplibre-gl-js-amplify
 ```
 
-Next, configure the Amplify library in your app. All geo-related configuration information is available within the `aws-exports.js` file. Add the following to your app's entry point (i.e. App.js):
+Import the library into your application:
 
 ```javascript
-import Amplify from 'aws-amplify';
 import { AmplifyMapLibreRequest } from "maplibre-gl-js-amplify";
-import awsconfig from './aws-exports';
-
-Amplify.configure(awsconfig);
```

Comment:
Do we need `maplibre-gl` to be imported at this point of the guide?

Reply:
No, because `maplibre-gl-js-amplify` uses it under the hood and we don't need to import it directly.